### ISSUE TITLE
Add Docker support 

### ISSUE THREAD

**Issue by INF800** [anarchy-ai/LLM-VM#139]

**Comment by mmirman** [comment]
This implies a new project for devops.  Be careful working on this issue unless you have great ideas!  Post them here

**Comment by mmirman** [comment]
Put it in dockerhub!

**Comment by prateek041** [comment]
I have some experience with k8s, so what is the actual scope of this issue ?

**Comment by cartazio** [comment]
I don’t think it makes sense to dockerize atm. This is more a library than an application.

**Comment by mertbozkir** [comment]
this issue resolved?

**Comment by mertbozkir** [comment]
<img width="1278" alt="image" src="https://github.com/anarchy-ai/LLM-VM/assets/63906053/28716af0-f2f1-4762-8065-2387ee47c12b">

[I published the llm-vm image on the docker hub! 🤞](https://hub.docker.com/repository/docker/19360859007/llm-vm/general)

and cool, because I can work on the #140 right now! 💪
